qa/suites/rgw/multisite/tasks/test_radosgw_admin_rest.yaml

```diff
@@ -0,0 +1,17 @@
+os_type: centos
+
+tasks:
+- install:
+- ceph: {cluster: c1}
+- ceph: {cluster: c2}
+- rgw:
+    c1.client.0:
+      port: 8000
+    c1.client.1:
+      port: 8001
+    c2.client.0:
+      port: 8000
+    c2.client.1:
+      port: 8001
+- radosgw_admin_rest:
+    endpoint: c2.client.0
```

Comment:
this yaml file is deploying two new ceph clusters and starting four new radosgw's without running the `rgw-multisite` task to configure multisite there

if you add this `radosgw_admin_rest` task to the end of `tasks/test_multi.yaml` instead, it will run against the existing multisite configuration

Reply:
My bad, I ran an incomplete task without checking the run results properly.